Feature/add project introduction (#436)
hpi-sam/digital-fuesim-manv: README.md

# Digitale FüSim MANV

> This is the codebase for a digital implementation of the FüSim MANV (Führungssimulation Massenanfall von Verletzen), a German simulation for training medical leaders on how to manage [Mass Casualty Incidents](https://en.wikipedia.org/wiki/Mass-casualty_incident).

Reviewer: ```suggestion
This is the codebase for a digital implementation of the "FüSim MANV" (Führungssimulation Massenanfall von Verletzen), a German simulation system for training of emergency medical services leadership personnel how to manage [Mass Casualty Incidents](https://en.wikipedia.org/wiki/Mass-casualty_incident).
```

**You can try it out at [https://fuesim-manv.de/](https://fuesim-manv.de/)**.

> ![image](https://user-images.githubusercontent.com/18506183/172071147-24b9aabe-51ee-4105-a5a4-6cbf8063eece.png)

Reviewer: Is it common practice to host images this way on some GitHub server? or should the image be in the repo?

_A screenshot of a part of an MCI exercise with initially ca. 50 patients at the Brandenburg Gate._

The concept is as follows:

-   A _trainer_ creates an exercise, which consists of _patients_, _vehicles_, _viewports_, _transferPoints_ and other objects placed on a map.
-   _Participants_ can then join the exercise.
-   The _trainer_ can restrict the participants to a specific _viewport_. The _participant_ cannot move out of this area.

> -   _Vehicles_ (including _material_, _personnel_ and (sometimes) _patients_) can be transferred to other areas via _transferPoints_.

Reviewer: Either its "vehicles (**containing** personnel, ...)" or "**objects** (including vehicles, personnel, ...)"

-   After the exercise is started, _patients_ that are not adequately treated by _personnel_ and _material_ can deteriorate and die. The goal of the _participants_ is to prevent the _patients_ from dying and transport them to the _hospitals_. To do this effectively they have to communicate with each other (via real radio devices, or remote via third-party services) and make the right decisions.
-   Afterward, the exercise can be evaluated via statistics and a "time-travel" feature.

> This simulation has been designed in cooperation with the [BABZ](https://www.bbk.bund.de/DE/Themen/Akademie-BABZ/akademie-babz_node.html) of the [Federal Office of Civil Protection and Disaster Assistance Germany](https://www.bbk.bund.de/DE/Home/home_node.html) and [Malteser Berlin](https://www.malteser-berlin.de/).

Reviewer: ```suggestion
This simulation has been designed in cooperation with and with support from the [Federal Academy for Civil Protection and Civil Defence](https://www.bbk.bund.de/DE/Themen/Akademie-BABZ/akademie-babz_node.html) of the [Federal Office of Civil Protection and Disaster Assistance Germany](https://www.bbk.bund.de/DE/Home/home_node.html), who are the original copyright holders of the analog "FüSim MANV" simulation system, and the [Malteser Hilfsdienst e.V. Berlin](https://www.malteser-berlin.de/).
```

> This simulation has been designed in cooperation with the [BABZ](https://www.bbk.bund.de/DE/Themen/Akademie-BABZ/akademie-babz_node.html) of the [Federal Office for Civil Protection and Disaster Assistance Germany](https://www.bbk.bund.de/DE/Home/home_node.html) and [Malteser Berlin](https://www.malteser-berlin.de/). Do only use this simulation for training purposes.

Reviewer 2: Just asking: Is `Federal Office for Civil Protection and Disaster Assistance Germany` an official name?

Author: Almost! I used the wrong preposition...
![image](https://user-images.githubusercontent.com/18506183/172072835-74f873f7-cd13-41b4-9985-566794884621.png)
https://www.bbk.bund.de/EN/Home/home_node.html;jsessionid=70E565B1C6927A79CE997B68CF91FAB8.live342

> This simulation has been designed in cooperation with the [BABZ](https://www.bbk.bund.de/DE/Themen/Akademie-BABZ/akademie-babz_node.html) of the [Federal Office for Civil Protection and Disaster Assistance Germany](https://www.bbk.bund.de/DE/Home/home_node.html) and [Malteser Berlin](https://www.malteser-berlin.de/). Do only use this simulation for training purposes.

Reviewer 2: The sentence `Do only use this simulation for training purposes.` sounds a bit ... weird to me, to be honest. I'm not sure how to make it better, though. So it wouldn't be a deal-breaker for me.

Author: Actually, I can not remember writing this...

Author: Maybe GitHub copilot proposed this, and I just accepted without thinking about it? - Anyway, I removed it.

The simulation is implemented as a web application with an Angular frontend and NodeJS backend.

> This project is currently developed as a [bachelor project](https://hpi.de/en/studies/before-your-studies/degree-programs/bachelor.html) at the HPI. You can find the official project website [here](https://hpi.de/teaching/bachelor-projects/digitales-fuehrungssimulationstraining.html).

Reviewer 2: ```suggestion
This project is currently developed as a [bachelor project](https://hpi.de/en/studies/before-your-studies/degree-programs/bachelor.html) at the [HPI](https://hpi.de/). You can find the official project website [here](https://hpi.de/teaching/bachelor-projects/digitales-fuehrungssimulationstraining.html).
```
